moveit_planners/trajopt/CMakeLists.txt

```diff
@@ -7,30 +7,122 @@ if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  moveit_core
-  roscpp
-  pluginlib
+find_package(catkin REQUIRED
+    COMPONENTS
+      interactive_markers
+      moveit_core
+      moveit_visual_tools
+      moveit_ros_planning
+      moveit_ros_planning_interface
+      pluginlib
+      roscpp
+      rosparam_shortcuts
+
+      trajopt
 )
 
 catkin_package(
-  CATKIN_DEPENDS
-    moveit_core
+    INCLUDE_DIRS include
+    LIBRARIES
+      interactivity_utils
```

Comment:
interactive markers are not used, remove

Reply:
This hasn't been removed yet, and I can't find it being used in this PR still.